hwangsae/relay.h

```diff
@@ -25,6 +25,8 @@
 
 #include <glib-object.h>
 
+#include "common.h"
```

Comment:
Why you included common in `relay.h`? There's nothing in the interface of this compilation unit that depends on it. I'd move the include into `relay.c`.

Reply:
I use this approach to as it was the same I was planned for `record.c`, basically to be able to expose common functions like get `hwangsae_common_get_local_ip` to applications like `recorder-agent`, which basically needs to it to build the URL.
My intention was to avoid an additional include file in the application (if it needs to use these functions). Of course I don't have any problems in change this if you think it is better in other way.